s3/pkg/datastore/yig/factory.go

```diff
@@ -56,18 +68,17 @@ func (ydf *YigDriverFactory) Init() error {
 	}
 
 	// read common config settings
-	cc, err := config.ReadCommonConfig("/etc/yig")
+	/*cc, err := config.ReadCommonConfig("/etc/yig")
```

Comment:
Remove unused codes please.

Reply:
@sfzeng thanks for your suggestion, i have removed these unused codes, pls have a check again, thanks.